.github/workflows/run-security-scans.yml

```diff
@@ -150,26 +151,27 @@ jobs:
           fi
           echo "image_url=$image_digest_url" >> $GITHUB_OUTPUT
 
-      - name: Check if already attested by Trivy-Scan attestor
+      - name: Check if already attested by Trivy Scan attestor
         id: get-attestation
         run: |
-          attestation=$(gcloud container binauthz attestations list --project="$ATTESTOR_PROJECT_ID" --attestor="projects/$ATTESTOR_PROJECT_ID/attestors/Trivy-Scan" --artifact-url="${{ steps.image-digest-url.outputs.image_url }}")
+          attestation=$(gcloud container binauthz attestations list --project="$ATTESTOR_PROJECT_ID" --attestor="projects/$ATTESTOR_PROJECT_ID/attestors/trivy-scan-attestor" --artifact-url="${{ steps.image-digest-url.outputs.image_url }}")
           attestationoutput=$(printf '%s' $attestation | jq --raw-input --slurp '.' --join-output)
           echo "attestation=$attestationoutput" >> $GITHUB_OUTPUT
+          echo "$attestationoutput"
 
-      - name: Perform Trivy-Scan attestation
+      - name: Perform Trivy Scan attestation
         id: attest-kartverket-kontekst
         if: steps.get-attestation.outputs.attestation == ''
         run: |
           gcloud beta container binauthz attestations sign-and-create \
```

Comment:
Jeg blir litt forvirret hvorvidt binauth kommandoene er i alfa, beta eller prod stage av google apiet. 

Ser at kommandoen i forrige steg ikke har "beta" i seg: `gcloud container binauthz attestations list ...`

Reply:
https://cloud.google.com/sdk/gcloud/reference/beta/container/binauthz/attestations/sign-and-create

Vi bruker bare beta der vi kjører sign-and-create, ettersom dette er en funksjon vi ønsker å kjøre men den er kun i beta-stage per nå.

Reply:
hmm - for meg så ser det ut til at `attestations list` også ligger i beta i følge
https://cloud.google.com/sdk/gcloud/reference/beta/container/binauthz/attestations/list

Hvordan kan jeg se hva som er i beta stage og ikke?

Reply:
Jeg fant det ut. Nederst på siden under "NOTES" så listes det hvilke alternative kommandoer som eksisterer. 
For `list` så finnes det alternativer men ikke for `sign-and-create` :o)

Reply:
Jeppers, det stemmer :)